standard/namespace.lua

```diff
@@ -7,11 +7,21 @@
 --
 
 local Class = require('Module:Class')
+local FnUtil = require('Module:FnUtil')
+local Table = require('Module:Table')
 
 local Namespace = {}
 
 function Namespace.isMain()
 	return mw.title.getCurrentTitle():inNamespace(0)
 end
 
+Namespace.getIdsByName = FnUtil.memoize(function()
+	return Table.map(mw.site.namespaces, function(id, ns) return ns.name, id end)
+end)
```

Comment:
Can we not just make this a "standard" function?

Reply:
what do you have in mind?

Reply:
```
function getIdsByName(......
```
etc

Reply:
I can write it like this: 

```
function Namespace.getIdsByName()
	return Table.map(mw.site.namespaces, function(id, ns) return ns.name, id end)
end
Namespace.getIdsByName = FnUtil.memoize(Namespace.getIdsByName)
```

is that what you're looking for?

Reply:
Missed the memoize, it's okay.